server/room-battle.ts

```diff
@@ -1146,7 +1147,8 @@ export class RoomBattle extends RoomGame<RoomBattlePlayer> {
 			void this.stream.write(`>player ${slot} ` + JSON.stringify(options));
 			if (playerOpts) player.hasTeam = true;
 
-			this.room.add(`|player|${slot}|${player.name}|${user.avatar}`);
+			this.room.add(`|player|${slot}|${player.name}|${user.avatar}|`);
```

Comment:
why the extra pipe at the end

Reply:
I had meant to hijack |player| for this originally. When I changed that, I left the pipe there, since I couldn't remember if it had been there before or not.